include/bbp/sonata/report_reader.h

```diff
@@ -127,8 +127,19 @@ class SONATA_API ReportReader
          * Return true if the data is sorted.
          */
         bool getSorted() const;
+
+        /**
+         * Return all the node ids.
+         */
         std::vector<NodeID> getNodeIds() const;
 
+        /**
+         * Return selected ids.
```

Comment:
I think we should pick a more descriptive name, and describe it more accurately.  Something like `getNodeElementMapping` or `getNodeIdElementIdMapping`, thoughs?

Reply:
I'd be for `getCompartmentIds`?

It would also be useful to see what the function is used for, that should be noted.

Reply:
not everything is a compartment, and the doc calls things [node element recordings](https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md#frame-oriented-node-element-recordings), so I think we should use that terminology.

Reply:
Should that also entail renaming the `CompartmentId` datatype definition, then?

Reply:
> renaming CompartmentId datatype definition

That's already part of the interface, so I think that ship has sailed.

Reply:
Alias & deprecate could be done already, though?

Reply:
I will defer to the people who maintain the `libsonata` reporting parts.